justfile

```diff
@@ -44,7 +44,7 @@ recreate:
     @just up "--force-recreate --build"
 
 # Show logs of all, or named, Docker services
-logs services="" args="-f":
+logs services="" args="":
```

Comment:
I don't love this change, as it'll mean having to specify `-f` every time I run this locally :confused: 

Could we perhaps have this function similar to the `IS_PROD` flag, maybe with `IS_CI`, and that automatically removes the `-f`?

Reply:
Yeah, that'd be good! We could use the same IS_CI flag to automatically add `-T` to the other docker-compose commands as well to avoid having to remember that when adding new actions :tada:

Reply:
That's a great point!

Reply:
@sarayourfriend if you're okay with it, I can work on adding `IS_CI` to this branch :slightly_smiling_face: